graylog2-server/src/test/java/org/graylog2/plugin/MessageSummaryTest.java

```diff
@@ -91,7 +91,7 @@ public void testGetFields() throws Exception {
 
     @Test
     public void testJSONSerialization() throws Exception {
-        final ObjectMapper mapper = new ObjectMapper();
+        final ObjectMapper mapper = new ObjectMapper().registerModule(new JodaModule());
```

Comment:
hmm, the test passes without this change. but wouldn't it be better if all our tests used the objectmapperprovider instead?

Reply:
> the test passes without this change
that was a mistake. it doesn't pass. still using the objectmapper is better :+1: 
it actually exposes an error in the test now. it should serialize as `stream_ids` which is correct